### DISCUSSION TITLE
Add “Next” button when viewing a notification

### DISCUSSION THREAD

**Discussion by alexec** [community/community#16046] [Projects and Issues]
I want to step through all my notifications. When viewing a notification (i.e. clicking into the issue or PR) there is a great toolbar at the top of the screen, it has buttons like “Done” which take you back to the list of notifications. 

To get to the next notification, I currently have to click “Done” and choose which one I’d like to read next.

I would like a “Next” button which takes me to the next unread notification. I like to read newest notifications first.

**Comment by ptere** [comment]
Would also be great if there were `[`/`]` keyboard shortcuts for "Done & Next," similar to Gmail.
